### ISSUE TITLE
3D model rendering on iOS 15

### ISSUE THREAD

**Issue by Bketting** [google/model-viewer#3102]
### Description

When loading a 3D model on an iOS 15 device, it turns out completely silver. On iOS 14 devices, the problem did/does not occur. The problem only occurs for a .glb that includes an MRO texture. See the photo below.

![IMG_5421](https://user-images.githubusercontent.com/45426768/148226938-8292521b-17b2-4fa3-98b1-f2a721d98da5.jpeg)

#### Live Demo
https://floatscans.com/models/aronson/

### Version
https://unpkg.com/@google/model-viewer@1.5.0/dist/model-viewer.js

### Browser Affected

- X Chrome, version: 96.0.4664.116
- [ ] Edge
- [ ] Firefox
- [ ] IE
- X Safari

### OS

- [ ] Android
- X iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

AR functions completely fine.

**Comment by elalish** [comment]
Since this is  clearly a Quick Look regression in Safari,  please file this bug on WebKit: https://bugs.webkit.org/

**Comment by fivelive** [comment]
Hey @Bketting,

Try using 2K textures instead of 4K.
It solved the issue for me in IOS15. (Took me a long time to figure out though)

You can use GLTF Pipeline to do so.

1) `gltf-pipeline -i model.glb -o model.gltf -t`
2) Change textures resolution to 2K
3) `gltf-pipeline -i model.gltf -o model-2k.glb`

(Here is the output `model-2k.glb` I got : https://file.io/j62DixkzRcAO)
You should not face that issue with that 2K version now.

Please let me know if that worked for you. 
I'm curious to know if that solves the issue for most people.

Great looking model by the way!

Best,

**Comment by Bketting** [comment]
> Since this is clearly a Quick Look regression in Safari, please file this bug on WebKit: https://bugs.webkit.org/

I flagged it with the WebKit team, still waiting for a solution

**Comment by Bketting** [comment]
> Hey @Bketting,
> 
> Try using 2K textures instead of 4K. It solved the issue for me in IOS15. (Took me a long time to figure out though)
> 
> You can use GLTF Pipeline to do so.
> 
> 1. `gltf-pipeline -i model.glb -o model.gltf -t`
> 2. Change textures resolution to 2K
> 3. `gltf-pipeline -i model.gltf -o model-2k.glb`
> 
> (Here is the output `model-2k.glb` I got : https://file.io/Xge5UmFe7ZJ8) You should not face that issue with that 2K version now.
> 
> Please let me know if that worked for you. I'm curious to know if that solves the issue for most people.
> 
> Great looking model by the way!
> 
> Best,

You’re right, the problem doesn’t occur with 2K textures. This only partly helps though, because I also need the 4K textures to work!
